server/game/ongoingEffects/OngoingEffectLibrary.ts

```diff
@@ -122,7 +122,11 @@ export = {
     // increaseLimitOnPrintedAbilities: (abilities) =>
     //     OngoingEffectBuilder.card.static(EffectName.IncreaseLimitOnPrintedAbilities, abilities),
     // loseAllNonKeywordAbilities: () => OngoingEffectBuilder.card.static(EffectName.LoseAllNonKeywordAbilities),
-    // loseKeyword: (keyword) => OngoingEffectBuilder.card.static(EffectName.LoseKeyword, keyword),
+    loseKeyword: (keywordOrKeywordProperties: KeywordNameOrProperties) =>
```

Comment:
Same note here as for SpecForce Soldier - let's comment this back out until we do the full implementation